content/en/getting_started/synthetics/api_test.md

```diff
@@ -32,8 +32,8 @@ In this example, an API test is created to ensure your website is constantly up
 
     - Add the URL of the endpoint you want to monitor. If you don’t know what to start with, you can use `https://www.shopist.io/`, which is a test web application.
     - Select **Advanced Options** to use custom request headers, authentication credentials, body content, or cookies.
-    - Add some tags to help organize and filter tests.
-    - Select locations for testing.
+    - You can set tags such as `prod` and `shopist` on your test. Tags will help to keep things organized and allow you to quickly find the tests you're interested in on the homepage.
+    - Choose from the **managed locations** to run your test from.
     - Click the **Test URL** button.
 
 {{< img src="getting_started/synthetics/api-test-config.png" alt="API test configuration"  style="width:60%;">}}
```

Comment:
Ideally we would change that screenshot to reflect the env tag that is now first class citizen on the creation form and the 15 locations we now have. And I think we could set the width to 100% for all screenshots/videos.